framework/validators/DefaultValueValidator.php

```diff
@@ -37,14 +37,23 @@ class DefaultValueValidator extends Validator
      */
     public $skipOnEmpty = false;
 
+    /**
+     * @var bool whether the [[value]] is callback. Default is `false`
+     *
+     * If `$callableValue` set `true`, [[value]] will be used as callback.
```

Comment:
`whether the [[value]] is callback.` and `If `$callableValue` set `true`, [[value]] will be used as callback.`  are not equal?

Reply:
So, the second sentence is redundant and I will remove it.